features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageWireframeHelper.kt

```diff
@@ -65,7 +76,57 @@ internal class ImageWireframeHelper(
         return imageWireframe
     }
 
-    private companion object {
-        private const val DRAWABLE_CHILD_NAME = "drawable"
+    @Suppress("NestedBlockDepth")
+    internal fun createCompoundDrawableWireframes(
+        view: TextView,
+        mappingContext: MappingContext,
+        startIndex: Int
+    ): MutableList<MobileSegment.Wireframe> {
+        val result = mutableListOf<MobileSegment.Wireframe>()
+        var currentIndex = startIndex
+        val density = mappingContext.systemInformation.screenDensity
+
+        view.compoundDrawables.forEachIndexed { index, _ ->
+            val drawable = view.compoundDrawables[index]
+
+            if (drawable != null) {
+                val drawableCoordinates = viewUtilsInternal.resolveCompoundDrawableBounds(
+                    view = view,
+                    drawable = drawable,
+                    pixelsDensity = density,
+                    index = index
+                )
+
+                val drawableX: Long? = drawableCoordinates?.x
+                val drawableY: Long? = drawableCoordinates?.y
+
+                if (drawableX != null && drawableY != null) {
+                    @Suppress("ThreadSafety") // TODO REPLAY-1861 caller thread of .map is unknown?
+                    createImageWireframe(
+                        view = view,
+                        index = ++currentIndex,
+                        x = drawableX,
+                        y = drawableY,
+                        width = drawable.intrinsicWidth
+                            .densityNormalized(density).toLong(),
+                        height = drawable.intrinsicHeight
+                            .densityNormalized(density).toLong(),
+                        drawable = drawable,
+                        shapeStyle = null,
+                        border = null
+                    )?.let { resultWireframe ->
+                        result.add(resultWireframe)
+                    }
+                }
+            }
+        }
+
+        return result
+    }
+
+    internal companion object {
+        @VisibleForTesting internal const val DRAWABLE_CHILD_NAME = "drawable"
+
+        @VisibleForTesting internal const val DRAWABLE_MIN_DP = 10
```

Comment:
I've set this as 10 - does this seem reasonable?

Reply:
hmm should we add a limit at all ?
I would start without actually as there is no content image in a TextView drawable usually. What do you think ?

Reply:
alright, I'll remove this for now and set to <= 0 and later on we can set a limit if needed